src/widgets/courses-school/ui/courses.tsx

```diff
@@ -12,11 +13,17 @@ import styles from './courses.module.scss';
 
 const cx = classNames.bind(styles);
 
+const MAX_COURSE_COUNT = 5;
```

Comment:
Please, place constant to `courses-school -> model -> constants.ts`


Accordint to docs

https://feature-sliced.design/docs/guides/issues/desegmented#solution

![image](https://github.com/user-attachments/assets/5da474c1-efc3-4cf7-9a61-fe61623987fd)

Reply:
This one used only in this widget, not sure that we need to move it to entities

Reply:
![image](https://github.com/user-attachments/assets/0ff2b50b-4b18-4038-84f4-c96766d8d050)

Reply:
Ignore this warn for now, it's a bug

https://github.com/feature-sliced/eslint-config/issues/104

Reply:
done